circle_evolution/species.py

```diff
@@ -32,7 +32,8 @@ def __init__(self, size, genes=128, genotype=None):
             genotype (np.ndarray): optional - initializes Specie with given genotype.
         """
         self.size = size
-        self.genotype = genotype if genotype is not None else np.random.rand(genes, 5)
+        self.genotypeWidth = 5 if len(size) < 3 else 7
```

Comment:
Can we work with underscored names instead of camelCase? Just for consistency for our modules.